frontend/src/core/config/config-schema.ts

```diff
@@ -144,6 +144,7 @@ export const UserConfigSchema = z
             }
             return width;
           }),
+        locale: z.string().nullable().optional(),
```

Comment:
Not sure if it would be worth doing a transformation to a `TypedString` type if we don't want to actually validate but make sure it's coming from our config?

```ts
locale.string().nullable().optional().transform((v) => {
  if (v == null) return undefined; // normalize null | undefined
  return v as LocaleString;
})
```

Reply:
i thought about making it a typed string. but i would need to wrap a lot of the react-aria components (or maybe i can do module augmentation), will give it a shot in a followup